### PR TITLE
chore: Fix some documentation issues

### PR DESCRIPTION
**Pull request by k-taro56**
## ☕️ Reasoning

Fix some documentation issues.

- Fix `import Auth from "@auth/core"`
- Remove `Provider` suffix
- Remove `title="pages/api/auth/[...nextauth].ts"`
- `next-auth` to `@auth/core`
- `js` to `ts` (but not all)
- `pnpm format`
- etc.

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [x] Ready to be merged

## 🎫 Affected issues

No Issue

## 📌 Resources

- [Security guidelines](https://github.com/nextauthjs/.github/blob/main/SECURITY.md)
- [Contributing guidelines](https://github.com/nextauthjs/.github/blob/main/CONTRIBUTING.md)
- [Code of conduct](https://github.com/nextauthjs/.github/blob/main/CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: packages/core/src/types.ts [hunk 1]
```diff
@@ -25,7 +25,7 @@
  *
  * @example
  * ```ts title=index.ts
- * import { Auth } from "@auth/core"
+ * import Auth from "@auth/core"
```
Comment (ndom91) [inline packages/core/src/types.ts:28]: Doesn't look like the `packaegs/core/src/index.ts` has a default export. I think this one was correct as-is

**Comment by ndom91** [comment]
Small nit about the `import { Auth } from "@auth/core"` lines (including in the `packages/core/src/index.ts` file).

But otherwise, looks good! Thanks!

File: packages/core/src/types.ts (hunk 1 above)
Comment (k-taro56) [reply packages/core/src/types.ts]: Done!

**Review by ndom91 — APPROVED** [review]
Awesome, thanks!

**Pull request merged**